src/domain/ledger/index.ts

```diff
@@ -31,8 +31,8 @@ export const ExtendedLedgerTransactionType = {
   LnIntraLedger: "ln_on_us",
 } as const
 
-export const toWalletId = (accountId: LiabilitiesAccountId): WalletId | null => {
-  const path = accountId.split(":")
+export const toWalletId = (walletIdPath: LiabilitiesWalletId): WalletId | null => {
```

Comment:
I thought we agreed to go with `CoreAccount` and `LiabilitiesAccount` but here its still `LiabilitiesWallet`? Is this pending a follow up PR?

Reply:
yes